runtime/src/main/java/org/corfudb/runtime/collections/CorfuQueue.java

```diff
@@ -285,23 +275,23 @@ public int compareTo(CorfuQueueRecord<? extends E> o) {
      * @return all the entries in the Queue
      */
     public List<CorfuQueueRecord<E>> entryList() {
-        return this.entryList(0L, MAX_INDEX_ENTRIES);
+        return this.entryList(new CorfuRecordId(0,0,0), MAX_INDEX_ENTRIES);
```

Comment:
Is it better to use invalid epoch and sequence instead of 0 ? Same for the rests

Reply:
done. thanks